### PR TITLE
bpo-33152

### PR DESCRIPTION
**Pull request by Windsooon**
clean code

https://bugs.python.org/issue33152

### CODE REVIEW

**Comment by the-knights-who-say-ni** [comment]
Hello, and thanks for your contribution!

I'm a bot set up to make sure that the project can legally accept your contribution by verifying you have signed the [PSF contributor agreement](https://www.python.org/psf/contrib/contrib-form/) (CLA).

Unfortunately we couldn't find an account corresponding to your GitHub username on [bugs.python.org](https://bugs.python.org/) (b.p.o) to verify you have signed the CLA (this might be simply due to a missing "GitHub Name" entry in your b.p.o account settings). This is necessary for legal reasons before we can look at your contribution. Please follow [the steps outlined in the CPython devguide](https://devguide.python.org/pullrequest/#licensing) to rectify this issue.


Thanks again to your contribution and we look forward to looking at it!

File: Lib/timeit.py [hunk 1]
```diff
@@ -199,11 +199,7 @@ def repeat(self, repeat=default_repeat, number=default_number):
         interested in.  After that, you should look at the entire
         vector and apply common sense rather than statistics.
         """
-        r = []
-        for i in range(repeat):
-            t = self.timeit(number)
-            r.append(t)
-        return r
+        return [self.time(number) for i in range(repeat)]
```
Comment (cryvate) [inline Lib/timeit.py:202]: self.time -> self.timeit

Comment (Windsooon) [reply Lib/timeit.py:202]: Sorry for the stupid mistake. :<

**Comment by methane** [comment]
FYI, you can skip creating issue for such small code cleanup

**Comment by Windsooon** [comment]
ok thank you for reviewing.

From Windson Yang

在 2018年3月27日，下午5:19，INADA Naoki <notifications@github.com<mailto:notifications@github.com>> 写道：


FYI, you can skip creating issue for such small code cleanup

**Comment by merwok** [comment]
I disagree with methane :)  In general, changes to CPython are bug fixes or new features.  Cleanups and refactorings are not made for themselves, but to ease the fixes or features.

Reasons are:
* minimize risk of new bugs
* make history / annotate more useful (we can see who thought about the fix / feature, not who moved things around)
* minimize churn (all commits are tested, reviewed, etc)

That said, thanks for your interest in improving Python!  I recommend you read https://devguide.python.org/ to learn more about CPython development and find a bug of feature to work on :)

**Pull request closed without merging**